metadata-db-app/src/cmr/metadata_db/services/concept_service.clj

```diff
@@ -629,12 +645,20 @@
           concept-type (:concept-type concept)
           concept-id (:concept-id concept)
           revision-id (:revision-id concept)]
+      ;; validate variable name
+      (when (and (= :variable concept-type)
+                 (> revision-id 1))
+        (let [previous-concept (c/get-concept db concept-type provider concept-id (- revision-id 1))]
```

Comment:
It would be better if we weren't relying on revision-id to always increment by 1.

I tried to find how we do this elsewhere and found an example - we do this:
`(mdb/find-collections context (assoc params :latest true))`

Reply:
Good point! We can call `c/get-concept` without the `revision-id` to get the latest concept, the `mdb/find...` call is not intended to be called within metadata db itself. Thanks!